spec/intro/2_Architecture.md

```diff
@@ -1,29 +1,31 @@
 # Architecture Overview
-Web3API's high-level architecture, as it pertains to Web3API-enabled applications, is as follows:
+Web3API's high-level architecture (as it pertains to Web3API-enabled applications)
 
 ![Architecture](../assets/Architecture.png)
 
 ## Components
 
 - [Web3API Client](../components/Web3API_Client.md) - Execution Environment  
-- [Web3API URI](../components/Web3API_URI.md) - Universal Identifier  
+- [Web3API URI](../components/Web3API_URI.md) - Uniform Resource Identifier (URI)  
 - [Web3API Package](../components/Web3API_Package.md) - Descriptive & Executable Resources
 - [Web3API Plugins](../components/Web3API_Plugins.md) - Non-WASM Based Web3APIs
 - [Web3API WASM Protocol](../components/Web3API_WASM_Protocol.md) - { Client <> WASM Module } Communication Protocol
 
 ## Step-By-Step Walkthrough
 
 ### **0: Initialize**  
-Firstly, applications must have a [Web3API Client](../components/Web3API_Client.md) bundled within their applications. When creating a Web3API Client instance, you may [provide additional URI redirects and plugins](TODO). Web3API Client's should provide the recommended [default redirects and plugins](TODO), enabling out of the box usability.  
+Applications must first include a [Web3API Client](../components/Web3API_Client.md) library.  
+When creating a Web3API Client instance, [additional URI redirects and plugins](TODO) may be provided.  
+Web3API Clients provide the recommended [default redirects and plugins](TODO), enabling out of the box usability.  
 
 ### **1: Query an API**  
-Applications can query an API by specifying what API to query using a [Web3API URI](../components/Web3API_URI.md), along with a GraphQL query.
+Applications query an API by specifying the API to query at a [Web3API URI](../components/Web3API_URI.md) and providing a GraphQL query.
```

Comment:
"Applications query APIs by specifying the API's [URI](../components/Web3API_URI.md), and providing a GraphQL query."